src/blocks/image.js

```diff
@@ -0,0 +1,19 @@
+import { propertiesToText } from './common/base';
+
+const imageToMjml = (item) => {
+    const mjmlProperties = item.properties
+    const properties = propertiesToText(mjmlProperties);
+
+    return `<mj-image ${properties} />`;
+};
+
+const imageToText = (item) => {
+    const alt = item.properties.alt || 'image';
+    const href = item.properties.href ? `(${item.properties.href})` : '';
+    return `[${alt}]${href}`;
```

Comment:
We have the link if [] for the others.

Reply:
I commit some changes to follow the markdown syntax.

Reply:
I'm not sure what do you mean with adding new lines.